scripts/globals/items/amanomurakumo.lua

```diff
@@ -1,13 +1,90 @@
 -----------------------------------------
--- ID: 18318
+-- ID: 18318, 18319, 18646, 18660, 18674, 19755, 19848, 21015, 21016, 21954
 -- Item: Amanomurakumo
 -- Additional Effect: 10% Attack Down
 -----------------------------------------
 require("scripts/globals/status");
 require("scripts/globals/magic");
 require("scripts/globals/msg");
+require("scripts/globals/weaponskills");
+require("scripts/globals/weaponskillids");
 -----------------------------------
 
+NAME_WEAPONSKILL = "AFTERMATH_AMANO";
```

Comment:
these should be local vars rather than global vars

Reply:
referenced in multiple places. Only one place to edit if needs changing?

Reply:
oh you mean local. derp thought I fixed that.

Reply:
I just mean making it local NAME_WEAPONSKILL instead of NAME_WEAPONSKILL so that the variable doesn't end up in the global namespace (like how aftermathTable is local)

Reply:
I did it in mandau.lua and that was it. -_-

Reply:
Aaaaand fixed.